site/content/specs.md

```diff
@@ -113,7 +113,7 @@ Notes:
 | fd_write                |   ✅    | Rust,TinyGo,Zig |
 | path_create_directory   |   ✅    | Rust,TinyGo,Zig |
 | path_filestat_get       |   ✅    | Rust,TinyGo,Zig |
-| path_filestat_set_times |   ❌    |                 |
+| path_filestat_set_times |   ✅    |            libc |
```

Comment:
what's the difference with libc vs Rust? I see libc is only here

Reply:
actually you are right, rust uses this. https://github.com/rust-lang/rust/blob/master/library/std/src/sys/wasi/fd.rs#L196

Reply:
cited